projects/storefrontlib/src/cms-components/myaccount/order/amend-order/amend-order-actions/amend-order-actions.component.ts

```diff
@@ -12,9 +14,23 @@ import {
 })
 export class AmendOrderActionsComponent {
   @Input() orderCode: string;
-  @Input() isValid: string;
+  @Input() form: FormGroup;
   @Input() backRoute: string;
   @Input() forwardRoute: string;
 
   @HostBinding('class') styles = 'row';
+
+  constructor(protected routingService: RoutingService) {}
+
+  continue(event: any): void {
```

Comment:
Please provide a type form this event. (`Event`, `MouseEvent`, etc) - just pick a correct one